axelor-human-resource/src/main/resources/views/ProjectPlanningTime.xml

```diff
@@ -264,6 +273,21 @@
       expr="eval: &quot;self.id IN (${project?.siteSet?.collect{it.id}.join(',')})&quot;"/>
   </action-attrs>
 
+  <action-attrs name="action-project-planning-time-attrs-project-task-default">
+    <attribute name="value" for="projectTask"
+      expr="eval: __repo__(ProjectTask).find(_projectTask?.id)"/>
+    <attribute name="readonly" for="projectTask" expr="eval: true"/>
+    <attribute name="value" for="timeUnit,displayTimeUnit"
+      expr="eval: __repo__(ProjectTask).find(_projectTask?.id)?.timeUnit"/>
+    <attribute name="value" for="project" expr="eval: _projectTask?.project"/>
+    <attribute name="value" for="employee" expr="eval: __user__?.employee"
+      if="__user__ in __repo__(Project).find(_projectTask?.project?.id)?.membersUserSet"/>
+    <attribute name="value" for="product" expr="eval: __user__?.employee?.product"
+      if="__user__ in __repo__(Project).find(_projectTask?.project?.id)?.membersUserSet &amp;&amp; __user__?.employee"/>
+    <attribute name="value" for="startDateTime" expr="eval: _sprintFromDate"/>
+    <attribute name="value" for="endDateTime" expr="eval: _sprintToDate"/>
```

Comment:
Remove the last 2 attributes and create a new function that take the startDateTime and endDateTime from the projectTask's sprint and use this function at the end of the onNew and the projectTask's onChange